lib/crutches/string.ex

```diff
@@ -41,27 +38,27 @@ defmodule Crutches.String do
       ["UsersSection.Commission.Department", "users_section/commission/department"],
 
   """
-  defdelegate underscore(string), to: Mix.Utils
+  defdelegate underscore(string), to: Macro
```

Comment:
We can probably just get rid of these delegates and tests (and cut a new release), since they're not really adding anything compared to the stdlib